modules/preview-react/select/lib/SelectBase.tsx

```diff
@@ -178,10 +171,8 @@ export interface SelectBaseProps extends CoreSelectBaseProps, StyledType {
 }
 
 export const buttonBorderWidth = 0.0625;
-export const buttonDefaultWidth = `${(spaceNumbers.xxxl * 7) / 2}rem`;
```

Comment:
If I remember correctly, was this used elsewhere since it was exported? I just want to make sure that my thinking was right and remembering that it's used in more than one place outside of this file.

Reply:
hmm good point! I know it was used within this file, let me try a project wide search! Good find!

Reply:
I don't see it used anywhere in application projects

Reply:
Double checked, looks like it's not a used export in ghe!